run-test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+file=$1
+
+function timeit {
```

Comment:
why is the actual timing function defined in another shell script? it seems unnecessary when no other scripts depend on that script.

Reply:
to keep things modular. 
Also, `run.sh` is coupled to timing function. So, it would be better to have `run` and `run-test` both call `timeit.sh`